pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/service_proxy_transport.go

```diff
@@ -0,0 +1,49 @@
+package openshiftkubeapiserver
+
+import (
+	"net/http"
+
+	listersv1 "k8s.io/client-go/listers/core/v1"
+	restclient "k8s.io/client-go/rest"
+)
+
+// serviceCABundleRoundTripper uses its own RoundTripper configured with a CA bundle containing both
+// handlerCA and the SSCS CA bundle read from the SSCS CA bundle configMap.
+type serviceCABundleRoundTripper struct {
+	serverName string
+	handlerCA  []byte
+	lister     listersv1.ConfigMapLister
+}
+
+func (r *serviceCABundleRoundTripper) getServiceCABundle() string {
+	configMap, err := r.lister.ConfigMaps("openshift-service-cert-signer").Get("signing-cabundle")
+	if err != nil {
+		return ""
+	}
+	bundle, ok := configMap.Data["cabundle.crt"]
+	if !ok {
+		return ""
+	}
+	return bundle
+}
+
+func (r *serviceCABundleRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	combinedCA := make([]byte, len(r.handlerCA))
+	copy(combinedCA, r.handlerCA)
+	bundle := r.getServiceCABundle()
```

Comment:
If I'm reading it right, you're doing this on every request to the webconsole.  You can wire up to the configmap informer instead only create a new restclient when it changes to avoid rebuilding clients for every request.

Reply:
Hmm I guess this is a proxy and not a redirect.  So I guess it makes sense to use the informer.  Sorry for my poor advice @mrogers950...

I think you will want logic similar to what we already do for the web console config (it is in this same folder).